hazelcast/src/main/java/com/hazelcast/internal/diagnostics/MetricsPlugin.java

```diff
@@ -70,35 +70,38 @@ public long getPeriodMillis() {
 
     @Override
     public void run(DiagnosticsLogWriter writer) {
-        writer.startSection("Metrics");
         probeRenderer.writer = writer;
+        // we set the time explicitly so that for this particular rendering of the probes, all metrics have exactly
```

Comment:
It is related.

The idea is that the time can be used as a key to find the other elements for the same metric  name.

So imagine you did a grep on 'cluster.clock.clusterStartTime' and you find

```
25-7-2017 07:40:44 Metrics[cluster.clock.clusterStartTime=1,500,957,590,070]
```

Ok.. nice.. but what about the other metrics? Because you have the date-time, you can now easily find the following because they all share the same date-time 

```
25-7-2017 07:40:44 Metrics[client.endpoint.totalRegistrations=0]
25-7-2017 07:40:44 Metrics[cluster.clock.clusterStartTime=1,500,957,590,070]
25-7-2017 07:40:44 Metrics[cluster.clock.clusterTime=1,500,957,644,045]
25-7-2017 07:40:44 Metrics[cluster.clock.clusterTimeDiff=0]
25-7-2017 07:40:44 Metrics[cluster.clock.clusterUpTime=53,975]
25-7-2017 07:40:44 Metrics[cluster.clock.localClockTime=1,500,957,644,045]
````